app/lib/theme/widgets/ContextualHelp.tsx

```diff
@@ -0,0 +1,89 @@
+import { IconDefinition } from "@fortawesome/fontawesome-svg-core";
+import { faExclamationCircle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Tooltip from "@mui/material/Tooltip";
+import { BC_GOV_LINKS_COLOR, TASKLIST_GREY_BG_COLOR } from "../colors";
+
+/**
+ * @param {string} title - The title of the tooltip to be displayed (can be HTML)
+ * @param {IconDefinition} icon - The icon to display in the tooltip; defaults to faExclamationCircle
+ * @param {string} placement - The placement of the tooltip; defaults to "top"
+ * @param {string} label - The label of the tooltip to be used as the aria-labelledby attribute; this gets passed by the parent component(FieldLabel)
+ * @returns {ReactElement} - The tooltip component
+ * To use this component, simply add "ui:tooltip" object to the uiSchema of the field you want to add a tooltip to.
```

Comment:
This component will also be used on the diffs page, which doesn't use rjsf. It looks to me like we can use it like any react component (`<ContextualHelp />`), but want to check if you had any concerns with it being used that way

Reply:
If I understood your point correctly, We have this tooltip on the diffs page too:
![Screenshot 2023-05-29 at 4 26 06 PM](https://github.com/bcgov/cas-cif/assets/52381975/9610815c-9c5a-4b74-b728-e173cccda3c7)